info/flow_and_pressure.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Understanding Pressure and Flow in Espresso
+parent: Information
+nav_order: 4
+---
+
+# Understanding Pressure and Flow in Espresso  
+
+The most important thing to understand when thinking about pressure in espresso is that **input flow + resistance = pressure & output flow**, meaning pressure in espresso is a result of the finely ground coffee resisting water flow through it. This water flow can come from a pump, piston, or any hybrid of the two (see: Uniterra nomad). The more flow is supplied, the more pressure can be built given less resistance. Let us start by examining a pump machine.   
```

Comment:
The input flow + resistance = pressure & output flow is probably a bit misleading. Might be better to say that input flow and pressure are not independent quantities and are related by the resistance of the puck. Also, I would leave out the distinction between input flow and output flow and add a caveat above to say that this doc will mainly focus on the extraction phase of espresso rather than headspace fill or preinfusion.